tests/vitest-vscode.ts

```diff
@@ -0,0 +1,23 @@
+import { runInRepo, $ } from '../utils'
+import { RunOptions } from '../types'
+import process from 'node:process'
+
+export async function test(options: RunOptions) {
+	await runInRepo({
+		...options,
+		// repo: 'vitest-dev/vscode',
+		// https://github.com/vitest-dev/vscode/pull/276
+		repo: 'hi-ogawa/vitest-vscode',
+		branch: 'test-vscode-e2e',
```

Comment:
Now that https://github.com/vitest-dev/vscode/pull/276 is merged:

```suggestion
		repo: 'vitest-dev/vscode',
		branch: 'test-vscode-e2e',
```

Reply:
Thanks for the ping :+1: Updated and verified CI on my fork
https://github.com/hi-ogawa/vitest-ecosystem-ci/actions/runs/8248943030/job/22560322434#step:8:1